qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -74,8 +80,10 @@
     variants:
         - unix_socket:
         - tcp_socket:
+            memory_leak_check = no
```

Comment:
We do not need to define `memory_leak_check = no`,because the default value of memory_leak_check is no.
`params.get("memory_leak_check", "no") == "yes"`